warehouse/admin/flags.py

```diff
@@ -15,6 +15,14 @@
 from warehouse import db
 
 
+class AdminFlagValue:
```

Comment:
```suggestion
class AdminFlagValue(enum.Enum):
```
(plus the necessary import)

Reply:
I'm hesitant to make the suggested change. This will make the values as a type of `enum` object. In order to access the string value, you have to call  `AdminFlagValue.XXX.value`. This will be a big change.

Reply:
(Accidentally deleted the comment I left earlier)

I'm hesitant to make the suggested change. In order to access the value, you need to use call `AdminFlagValue.XXX.value`. There are many places which just expect a string, e.g tests/unit/admin/test_flags.py. However, it's a good idea and I'm happy to make the changes with another PR.

Reply:
I still think this should be an `Enum` to be consistent with other enum-like patterns we have, and it's fine to just include it in this PR as it's own commit.